packages/terra-form-select/src/_Frame.jsx

```diff
@@ -213,13 +256,40 @@ class Frame extends React.Component {
   /**
    * Opens the dropdown.
    */
-  openDropdown() {
+  openDropdown(event) {
     if (this.state.isOpen || this.props.disabled) {
       return;
     }
 
+    /**
+     * Avoids focusing the input if the toggle button is used to open the select menu.
+     * This is to avoid an issue with VoiceOver on iOS where shifting to toggle button while the
+     * input is focused / onScreen keyboard is open unexpected focus shift when the onScreen
+     * keyboard is closed
+     */
+    if (event && event.target
+      && (event.target.hasAttribute('data-terra-form-select-toggle-button')
+      || event.target.hasAttribute('data-terra-form-select-toggle-button-icon'))) {
+      this.setState({ isOpen: true, isPositioned: false });
+
+      // Allows time for state update to render select menu DOM before shifting focus to it
+      setTimeout(() => {
+        if (document.querySelector(this.selectMenu)) {
+          document.querySelector(this.selectMenu).focus();
+        }
+      }, 10);
```

Comment:
![gamble](https://user-images.githubusercontent.com/6720991/55102774-273a6b80-5095-11e9-8d9e-b4ff9de800a2.gif)

Reply:
Focus management is tricky.